[linux] Test app crashes with R and E commands

The current version of `glfw-app` in `master` crashes at least with two commands, `R` (enable the route demo) and `E` (insert an example building extrusion layer) with the following message:

```
mbgl-glfw: ../../../src/mbgl/style/expression/interpolate.cpp:271: mbgl::style::expression::Interpolate::Interpolate(const Type&, mbgl::style::expression::Interpolator, std::unique_ptr<mbgl::style::expression::Expression>, std::map<double, std::unique_ptr<mbgl::style::expression::Expression> >): Assertion `input->getType() == type::Number' failed.
Aborted (core dumped)
Makefile:364: recipe for target 'run-glfw-app' failed
make: *** [run-glfw-app] Error 134
```

I haven't tested the app thoroughly so it's possible that other commands are crashing as well.

Capturing chat with @ivovandongen , this looks like a recent regression as he didn't experience the issue when he last tried the app in https://github.com/mapbox/mapbox-gl-native/commit/b84a184d2ddc496908506c9c69ed6b4778213077.

cc: @mapbox/embedded

--- a/platform/glfw/glfw_view.cpp
+++ b/platform/glfw/glfw_view.cpp
@@ -640,7 +640,7 @@ void GLFWView::toggle3DExtrusions(bool visible) {
     extrusionLayer->setMinZoom(15.0f);
     extrusionLayer->setFilter(Filter(eq(get("extrude"), literal("true"))));
     extrusionLayer->setFillExtrusionColor(SourceFunction<mbgl::Color>(
-        interpolate(linear(), get("height"),
+        interpolate(linear(), number(get("height")),
                     0.f, toColor("#160e23"),
                     50.f, toColor("#00615f"),
                     100.f, toColor("#55e9ff"))));

